tests/system/Commands/SessionsCommandsTest.php

```diff
@@ -29,23 +31,11 @@ public function tearDown(): void
 	public function testCreateMigrationCommand()
 	{
 		command('session:migration');
-		$result = CITestStreamFilter::$buffer;
 
 		// make sure we end up with a migration class in the right place
 		// or at least that we claim to have done so
 		// separate assertions avoid console color codes
-		$this->assertStringContainsString('Created file:', $result);
-		$this->assertStringContainsString('_CreateCiSessionsTable.php', $result);
-	}
-
-	public function testOverriddenCreateMigrationCommand()
-	{
-		command('session:migration -t mygoodies');
-		$result = CITestStreamFilter::$buffer;
-
-		// make sure we end up with a migration class in the right place
-		$this->assertStringContainsString('Created file:', $result);
-		$this->assertStringContainsString('_CreateMygoodiesTable.php', $result);
+		$this->assertStringContainsString('_CreateCiSessionTable.php', CITestStreamFilter::$buffer);
```

Comment:
Reinstate the ability to override the sessions table and also this test. That feature was not introduced by Generator Command. It was before so removing that is a BC break.